avalon/tools/cbloader/model.py

```diff
@@ -75,7 +72,7 @@ def set_version(self, index, version):
             "version_document": version,
             "author": version_data.get("author", None),
             "time": version_data.get("time", None),
-            "family": version_data.get("families", ["<unknown>"])[0],
+            "families": version_data.get("families", ["<unknown>"])[0],
```

Comment:
Actually, the "families" for a subset is:

```
family = version_data.get("family", None)
families = list(version_data.get("families", []))
if family:

    # If `family` data it's supposed to be the 
    # primary family so we ensure it's first
    if family in families:
        families.remove(family)
    families.insert(0, families)
```

I think it must end up becoming something like this. 

Though, this will make things *a lot* trickier! Because the "family" field in the cbloader will then suddenly become a list as well, which will confuse artists. Then which will we actually show? I'm not sure whether including this fix in this PR is the best way going forward - it'll require quite some logic changes.

Note: currently we're only using a single family. Our productions don't use the multiple family aspect yet.

Reply:
I will reverse the `family` changes so we can take the time to implement it good.

Reply:
Created PLN-47 to remind us of this in the future. :)